Tests/SPIManifestTests/ManifestTests.swift

```diff
@@ -68,19 +102,21 @@ class ManifestTests: XCTestCase {
     func test_load_maxSize() throws {
         let data = Data("""
                 version: 1
+                metadata:
+                  authors: Christian Noon, Jon Shier and other contributors
```

Comment:
I don't think we need to modify this test as we add this key. The test checks that for a predefined set of data that the `maxByteSize` parameter of `load` succeeds/fails. We should look at how we call this in production to make sure that we have enough capacity for this extra data, but I don't think this test needs to change.